recipe/recipe.yaml

```
@@ -0,0 +1,264 @@
+      ignore_run_exports:
+        by_name:
+          - spdlog
+      build:
+        - ${{ compiler('cxx') }}
+        - ${{ stdlib("c") }}
+        - cmake
+        - ninja
+        - if: build_platform != target_platform
+          then:
+            - python
+            - cross-python_${{ target_platform }}
+            - pybind11
+            - pybind11-abi
+      host:
+        - python
+        - scikit-build
+        - pip
+        - setuptools <79
+        - pybind11
+        - pybind11-abi
+        - openssl
+        - yaml-cpp
+        - cpp-expected
+        - spdlog
+        - fmt >=11.1.0
+        - termcolor-cpp
+        - nlohmann_json <3.12
+        - ${{ pin_subpackage('libmamba', exact=True) }}
+        # zstd is not a direct dependency but this specification is needed to
+        # distinguish builds of libmambapy for specific specific version of zstd.
+        - zstd
+      run:
+        - python
+        - ${{ pin_subpackage('libmamba', exact=True) }}
+    tests:
+      - python:
+          imports:
+            - libmambapy
+            - libmambapy.bindings
+      - script:
+        - python -c "import libmambapy; assert libmambapy.__version__ == '${{ version }}'"
+
+  - package:
+      name: mamba
+    build:
+      script: build_mamba
+    requirements:
+      ignore_run_exports:
+        from_package:
+          - spdlog
+          - fmt
+          - libsolv
+          - ${{ compiler('cxx') if linux }}
+          - ${{ "python" if win }}
+          - if: unix
+            then:
+              - libcurl
+              - openssl
+      build:
+        - ${{ compiler('cxx') }}
+        - ${{ stdlib("c") }}
+        - cmake
+        - ninja
+        - if: win
+          then:
+          - vcpkg-tool
+          - python
+          - curl >=8.4.0
+          - zlib
+      host:
+        - cli11 >=2.2,<3
+        - reproc
+        - reproc-cpp
+        - ${{ pin_subpackage('libmamba', exact=True) }}
+        - spdlog
+        # zstd is not a direct dependency but this specification is needed to
+        # distinguish build of mamba for specific specific version of zstd.
+        - zstd
+      run:
+        - ${{ pin_subpackage('libmamba', exact=True) }}
+    tests:
+      - script:
+        - mamba --help
+        - if: unix
+          then:
+            - test -f "${PREFIX}/bin/mamba"
+            - test -f "${PREFIX}/condabin/mamba"
+            # TODO: remove `etc/profile.d/mamba.sh` after 30th Sept. 2025
+            - test -f "${PREFIX}/etc/profile.d/mamba.sh"
+            # Execute this script
+            - . ${PREFIX}/etc/profile.d/mamba.sh
+            - ${PREFIX}/condabin/mamba --help
+            - export MAMBA_ROOT_PREFIX="$(mktemp -d)"
+            - mamba clean --all --dry-run
+            # The following test is segfaulting but we couod not reproduce it outside the build
+            # environment, so we're moving on with the 2.0.0beta0 release.
+            # - mamba repoquery depends mamba
```

Unrelated: We could check whether we could remove this now.
```suggestion
            - mamba repoquery depends mamba
```